ethereum_history_api/circuits/lib/src/misc/uint256.nr

```diff
@@ -0,0 +1,50 @@
+use dep::std::ops::Add;
+
+global uint128_overflow_value = 340282366920938463463374607431768211456; // 2^128
+
+struct U256 {
```

Comment:
Let's keep U128 and U256 consistent and either use hi, lo or high, low in both

Reply:
I don't agree. In general - when a library uses a bad name - we want to fix it in our code. It seems to me that high is more readable than hi

Reply:
I don't like `hi` and `lo`...